pep/app/pods/components/search/item/bibliographic/template.hbs

```diff
@@ -12,25 +25,14 @@
         {{/if}}
     </div>
     <div class="pl-2 d-flex justify-content-end align-items-start">
-        {{#if @item.relatedrx}}
-            <span class="mr-2">
-                <FaIcon @icon="books" class="p-0 "/>
-                <EmberTooltip
-                    @text={{t "search.item.relatedDocuments"}}
-                    @popperContainer="body"
-                    @side="left"
-                    @spacing={{5}}
-                    @effect="slide"
-                    @innerClass=""
-                />
-            </span>
-        {{/if}}
+        {{!-- TODO: Why does using Button::Tooltip here not update the iconPrefix --}}
```

Comment:
When you use `Button::WithTooltip`, you'll need to update that wrapper component to pass `@iconPrefix` down into the inner `Button` component, i didn't get all of those args when i created it.